systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -130,60 +130,16 @@ protected HelmClient helmClient() {
         return kubeCluster().helmClient().namespace(getNamespace());
     }
 
-    static String kafkaClusterName(String clusterName) {
-        return KafkaResources.kafkaStatefulSetName(clusterName);
-    }
-
-    static String kafkaConnectName(String clusterName) {
-        return clusterName + "-connect";
-    }
-
-    static String kafkaMirrorMakerName(String clusterName) {
-        return clusterName + "-mirror-maker";
-    }
-
-    static String kafkaPodName(String clusterName, int podId) {
-        return KafkaResources.kafkaPodName(clusterName, podId);
-    }
-
-    static String kafkaServiceName(String clusterName) {
-        return KafkaResources.bootstrapServiceName(clusterName);
-    }
-
-    static String kafkaHeadlessServiceName(String clusterName) {
-        return KafkaResources.brokersServiceName(clusterName);
-    }
-
-    static String kafkaMetricsConfigName(String clusterName) {
-        return KafkaResources.kafkaMetricsAndLogConfigMapName(clusterName);
-    }
-
-    static String zookeeperClusterName(String clusterName) {
-        return KafkaResources.zookeeperStatefulSetName(clusterName);
-    }
-
-    static String zookeeperPodName(String clusterName, int podId) {
-        return KafkaResources.zookeeperPodName(clusterName, podId);
-    }
-
     static String zookeeperServiceName(String clusterName) {
-        return zookeeperClusterName(clusterName) + "-client";
+        return KafkaResources.zookeeperStatefulSetName(clusterName) + "-client";
     }
 
     static String zookeeperHeadlessServiceName(String clusterName) {
-        return zookeeperClusterName(clusterName) + "-nodes";
-    }
-
-    static String zookeeperMetricsConfigName(String clusterName) {
-        return KafkaResources.zookeeperMetricsAndLogConfigMapName(clusterName);
+        return KafkaResources.zookeeperStatefulSetName(clusterName) + "-nodes";
     }
 
     static String zookeeperPVCName(String clusterName, int podId) {
-        return "data-" + zookeeperClusterName(clusterName) + "-" + podId;
-    }
-
-    static String entityOperatorDeploymentName(String clusterName) {
-        return KafkaResources.entityOperatorDeploymentName(clusterName);
+        return "data-" + KafkaResources.zookeeperStatefulSetName(clusterName) + "-" + podId;
```

Comment:
Should we have a method on KafkaResources for getting the PVC name?

Reply:
Ye, might be useful. Guess we can add it in another PR and use it there.